packages/custom-elements-languageserver/package.yaml

```diff
@@ -1,24 +1,15 @@
 ---
 name: custom-elements-languageserver
 description: |
-  The best way to utilize the Custom Elements Language Server is to enable the Custom Elements Manifest (CEM) in your
-  project by installing a CEM generator like one provided by The Open WC Team.
-
-  Generating a CEM in watch mode will provide you with the best user experience. If your dependencies ship with a Custom
-  Elements Manifest, those will be utilized also.
+  Custom Elements Language Server provides useful language features for Web Components. Features include code actions, completons, diagnostics and more.
 homepage: https://github.com/Matsuuu/custom-elements-language-server
 licenses:
   - BSD-3-Clause
-languages:
-  - HTML
-  - JavaScript
-  - TypeScript
-  - JSX
 categories:
   - LSP
 
 source:
-  id: pkg:npm/custom-elements-languageserver@0.0.8
+  id: pkg:npm/custom-elements-languageserver@1.0.1
```

Comment:
Hm I wonder why Renovate hasn't picked this up. I believe it may be because the current version is `<1`, will look into it.

Reply:
I think it might be that I pushed them today too? Maybe it's just a bit slow. Or could be that it's a major version bump

Reply:
Ah yeah seems like [it was picked up ~15 hours ago](https://github.com/mason-org/mason-registry/pull/3393).